plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/LocalAppEngineServerWizardFragment.java

```diff
@@ -91,13 +90,12 @@ public void widgetSelected(SelectionEvent event) {
     return cloudSdkComposite;
   }
 
-  private boolean isCloudSdkConfigured() {
+  private String getCloudSdkLocation() {
     try {
       CloudSdk cloudSdk = new CloudSdk.Builder().build();
-      cloudSdkLocation = cloudSdk.getSdkPath().toString();
-      return true;
+      return cloudSdk.getSdkPath().toString();
```

Comment:
I'm confused. How does this pick up a non-standard location the user has set in the preferences?